sass/mixins/status/status.scss

```diff
@@ -0,0 +1,32 @@
+@import "palette.scss";
+
+@mixin status($type) {
+  display: inline-block;
+  padding: 7px 10px;
+  border-radius: 25px;
+  text-align: center;
+
+  &.alt {
```

Comment:
Hmm, not sure. "alt" is "style" related, but arguable so is "variant".

Would "alt" map to a state, like "active", "focussed", "disabled"? If so, I think a attribute is more appropriate.

If its style only, I think a skin class (like we discussed the other day) on a parent element or the elem itself could flip the colours of the elems